postlicyd/dns.h

```diff
@@ -89,7 +89,7 @@ typedef void (*dns_result_callback_f)(dns_result_t *result, void *data);
  */
 __attribute__((nonnull(1,3,4)))
 bool dns_resolve(const char *hostname, dns_rrtype_t type,
-                 ub_callback_t callback, void *data);
+                 ub_callback_type callback, void *data);
```

Comment:
You should provide a compatibility layer with the older versions of the lib in some way. For example:

```c
#define UNBOUND_FULL_VERSION  \
    ((UNBOUND_VERSION_MAJOR << 16) \
    | (UNBOUND_VERSION_MINOR << 8) \
    | (UNBOUND_VERSION_MICRO))

#if UNBOUND_FULL_VERSION < 0x010601
typedef ub_callback_t ub_callback_type;
#endif
```